test/suites/drift/suite_test.go

```diff
@@ -95,6 +95,10 @@ var _ = Describe("Drift", func() {
 			},
 		})
 		selector = labels.SelectorFromSet(dep.Spec.Selector.MatchLabels)
+
+		rawContent, err := os.ReadFile("testdata/al2023_userdata_input.yaml")
+		Expect(err).ToNot(HaveOccurred())
+		customUserData = lo.ToPtr(fmt.Sprintf(string(rawContent), env.ClusterName, env.ClusterEndpoint, env.ExpectCABundle()))
```

Comment:
is it overkill to marshal, merge, and unmarshal the userdata to do this rather than a magic ordering of string inputs? happy to do what you think is best here.

Reply:
I agree that seems like the better approach, this is something we do in other tests already though. Makes sense to me to make the change in a follow-up PR across the whole project.